package-system/OpenEXR/build_config.json

```diff
@@ -1,100 +1,83 @@
             ],
-            "cmake_generate_args_release":[
+            "custom_toolchain_file" : "../cmake/Platform/Mac/Toolchain_mac.cmake",
+            "cmake_generate_args_release": [
                "-G",
                "Xcode",
-               "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.13",
-               "-DOPENEXR_BUILD_PYTHON_LIBS=OFF",
-               "-DOPENEXR_BUILD_TESTS=OFF",
-               "-DOPENEXR_BUILD_UTILS=OFF",
-               "-DOPENEXR_BUILD_SHARED=OFF",
-               "-DOPENEXR_BUILD_STATIC=ON",
-               "-DCMAKE_INSTALL_PREFIX=\"../../ilmbase-mac/ilmbase\"",
-               "-DCMAKE_BUILD_TYPE=Release",
-               "-DZLIB_ROOT=../../../zlib-mac/zlib"
-            ]
+               "-DBUILD_TESTING=OFF",
+               "-DBUILD_SHARED_LIBS=OFF",
+               "-DOPENEXR_INSTALL_EXAMPLES=OFF",
+               "-DOPENEXR_INSTALL_TOOLS=OFF",
+               "-DDOCS=OFF",
+               "-DPYTHON=OFF",
+               "-DCMAKE_INSTALL_PREFIX=\"../../OpenEXR-mac/OpenEXR\"",
+               "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "custom_test_cmd" : [
+               "./test_openexr_mac.sh"
+			],
+			"extra_files_to_copy": [
+				["FindImath.cmake", "FindImath.cmake"]
+			]
          }
       },
       "Linux":{
          "Linux":{
-            "cmake_find_template":"Findilmbase.cmake.Linux",
-            "cmake_generate_args_debug":[
-               "-G",
-               "Ninja",
-               "-DOPENEXR_BUILD_PYTHON_LIBS=OFF",
-               "-DOPENEXR_BUILD_TESTS=OFF",
-               "-DOPENEXR_BUILD_UTILS=OFF",
-               "-DOPENEXR_BUILD_SHARED=OFF",
-               "-DOPENEXR_BUILD_STATIC=ON",
-               "-DCMAKE_INSTALL_PREFIX=\"../../ilmbase-linux/ilmbase\"",
-               "-DCMAKE_BUILD_TYPE=Debug",
-               "-DZLIB_ROOT=../../../zlib-linux/zlib"
-            ],
+            "build_configs" : ["Release"],
+            "depends_on_packages" :[ 
+                ["zlib-1.2.11-rev5-linux", "9be5ea85722fc27a8645a9c8a812669d107c68e6baa2ca0740872eaeb6a8b0fc", ""]
+           ],
             "cmake_generate_args_release":[
                "-G",
                "Ninja",
-               "-DOPENEXR_BUILD_PYTHON_LIBS=OFF",
-               "-DOPENEXR_BUILD_TESTS=OFF",
-               "-DOPENEXR_BUILD_UTILS=OFF",
-               "-DOPENEXR_BUILD_SHARED=OFF",
-               "-DOPENEXR_BUILD_STATIC=ON",
-               "-DCMAKE_INSTALL_PREFIX=\"../../ilmbase-linux/ilmbase\"",
-               "-DCMAKE_BUILD_TYPE=Release",
-               "-DZLIB_ROOT=../../../zlib-linux/zlib"
-            ]
+               "-DBUILD_TESTING=OFF",
+               "-DOPENEXR_RUN_FUZZ_TESTS=OFF",
+               "-DOPENEXR_INSTALL_EXAMPLES=OFF",
+               "-DOPENEXR_BUILD_TOOLS=OFF",
+               "-DBUILD_SHARED_LIBS=OFF",
+               "-DCMAKE_INSTALL_PREFIX=\"../../openexr-linux/OpenEXR\""
+            ],
+			"extra_files_to_copy": [
+				["FindImath.cmake", "FindImath.cmake"]
+			]
```

Comment:
nit: test alignment seems off

Reply:
Moved the extra_files_to_copy to global scope for this file.